src/pages/BasketPage/components/BasketPageContent/index.js

```diff
@@ -128,16 +91,9 @@ const BasketPageContent = () => {
   if (!data.isLoaded) {
     return null
   }
-  const basketEmpty =
-    groceriesBasketData.length === 0 &&
-    productsBasketData.length === 0 &&
-    dishesBasketData.length === 0 &&
-    savedListBasketData.length === 0
-
-  const messege = <Styles.Message>Your basket is empty!</Styles.Message>
 
-  return basketEmpty ? (
-    messege
+  return data.isBasketEmpty ? (
+    <Styles.Message>Your basket is empty!</Styles.Message>
   ) : editMode ? (
```

Comment:
staraj się nie zagniezdzac ternary w ternary, bo to malo czytelne. Tutaj mozesz to rozbic tak:
if(data.isBasketEmpty){
return  (
    <Styles.Message>Your basket is empty!</Styles.Message>
  )
}

return editMode ? ... : ...

Reply:
Poprawione.